dipy/reconst/shm.py

```diff
@@ -399,10 +399,10 @@ def real_sh_descoteaux_from_index(m_values, l_values, theta, phi, *, legacy=True
 
     Y_l^m =
     \begin{cases}
-        \sqrt{2} * \Im(Y_l^m) \; if m > 0 \\
-        Y^0_l \; if m = 0 \\
-        \sqrt{2} * \Re(Y_l^m)  \; if m < 0 \\
-    \end{cases}
+        \\sqrt{2} * \\Im(Y_l^m) \\; if m > 0 \\
```

Comment:
Double backslashes should not be necessary if we use the raw string marker for the docstring. Would you mind doing the latter in a separate PR so that it can be merged independently and all doc-related PRs that have been opened lately can be rebased with less work? Thanks Ariel.

Reply:
ruff added these automatically. Do we need to configure it not to do that?

Reply:
:upside_down_face: Weird that it did that now and not before, and only on these files. I think using r involves less work, and the code is more readable.